x-pack/plugins/infra/public/pages/logs/log_entry_categories/sections/top_categories/category_example_message.tsx

```diff
@@ -60,6 +108,38 @@ export const CategoryExampleMessage: React.FunctionComponent<{
           wrapMode="none"
         />
       </LogEntryColumn>
+      <LogEntryColumn {...columnWidths[iconColumnId]}>
+        {isHovered || isMenuOpen ? (
+          <LogEntryContextMenu
+            isOpen={isMenuOpen}
+            onOpen={openMenu}
+            onClose={closeMenu}
+            items={[
+              {
+                label: i18n.translate('xpack.infra.logs.categoryExample.viewInStreamText', {
+                  defaultMessage: 'View in stream',
+                }),
+                onClick: viewInStreamLinkProps.onClick!,
```

Comment:
Can we add the `href` attribute to this item as well to offer the things `onClick` doesn't by itself (`viewInStreamLinkProps` will have it already).